### PR TITLE
[11.0] date_range: Migration to 11.0

### PR DESCRIPTION
**Pull request by lmignon**
supersedes https://github.com/OCA/server-tools/pull/1017

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Functional review

**Comment by lmignon** [comment]
branch rebased to cleanup the history.

File: date_range/wizard/date_range_generator.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="date_range_generator_view_form" model="ir.ui.view">
+        <field name="name">date.range.generator.form</field>
+        <field name="model">date.range.generator</field>
+        <field name="arch" type="xml">
+            <form string="Genrate Date Ranges">
```
Comment (HviorForgeFlow) [inline date_range/wizard/date_range_generator.xml]: Miss spelling: Genrate <> Generate

**Review by HviorForgeFlow — APPROVED** [review]
Functional review :+1:

**Comment by etobella** [comment]
LGTM :+1:

**Comment by pedrobaeza** [comment]
@jbeficent next time give the opportunity to squash latest commit which is a fixup.

**Comment by JordiBForgeFlow** [comment]
@pedrobaeza oh, right!

**Pull request merged**